### PR TITLE
PS-5796 Document PS-5795:8.0 MyRocks memcmp-able key packing and comp…

### PR DESCRIPTION
**Pull request by patrickbirch**
…arison is not Unicode 9.0.0 compliant

Added section to myrocks/limitations.rst

### CODE REVIEW

File: doc/source/myrocks/limitations.rst [hunk 1]
```diff
@@ -90,3 +90,11 @@ You should also consider the following:
      |MySQL| Documentation: Preparing Your Installation for Upgrade
         https://dev.mysql.com/doc/refman/8.0/en/upgrade-prerequisites.html
 
+* |Percona Server| 8.0 and Unicode 9.0.0 standards have defined a change in the
+  handling of binary collations. These collations are handled as NO PAD,
+  trailing spaces are included in comparisons. A binary collation comparison
```
Comment (george-lorch) [inline doc/source/myrocks/limitations.rst]: in **key** comparisons.

File: doc/source/myrocks/limitations.rst [hunk 2]
```diff
@@ -90,3 +90,11 @@ You should also consider the following:
...
+  may result in two unique rows inserted and does not generate a`DUP_ENTRY`
+  error. MyRocks key encoding and comparison does not account for this
+  character set attribute. Older indexes, created under Version 5.7 and
+  early versions of 8.0 prior to 8.0.13-3 must be rebuilt to return
```
Comment (george-lorch) [inline doc/source/myrocks/limitations.rst]: Drop the "Older versions ..." as it will only apply once we actually fix this issue, which we haven't yet.

**Review by george-lorch — CHANGES_REQUESTED** [review]
two minor things, but otherwise is well stated.

File: doc/source/myrocks/limitations.rst (hunk 2 above)
Comment (patrickbirch) [reply doc/source/myrocks/limitations.rst]: Updated based on comments.

**Review by george-lorch — APPROVED** [review]
LGTM

**Pull request merged**